modules/grizzly/src/main/java/org/glassfish/grizzly/Connection.java

```diff
@@ -492,4 +492,16 @@ void setProcessorSelector(
         LOCALLY, REMOTELY
     }
 
+    /**
+     * Get ClassLoader. Default is null.
+     * @return
+     */
+     ClassLoader getClassLoader();
```

Comment:
it is still not clear what this is. Is this the classloader that loaded this class? the TCCL? What would someone use this for?